src/databricks/labs/ucx/cli.py

```diff
@@ -446,5 +446,26 @@ def migrate_tables(w: WorkspaceClient, prompts: Prompts, *, ctx: WorkspaceContex
             deployed_workflows.run_workflow("migrate-external-hiveserde-tables-in-place-experimental")
 
 
+@ucx.command
+def migrate_dbsql_dashboards(w: WorkspaceClient, dashboard_id: str | None = None):
+    """Migrate table references in DBSQL Dashboard queries"""
+    ctx = WorkspaceContext(w)
+    ctx.redash.fix_dashboards(dashboard_id)
```

Comment:
```suggestion
    ctx.redash.migrate_dashboards(dashboard_id)
```

Is it migrate or fix? For tables we use migrate